tests/cypress/fixtures/createSite.groovy

```diff
@@ -0,0 +1,15 @@
+import org.jahia.services.sites.JahiaSite
```

Comment:
Don't think this script is needed as it looks like we're already using the one from `@jahia/cypress`